### PR TITLE
Updates project to common catalog structure

### PR DESCRIPTION
**Pull request by m4rkmckenna**
* Add karaf feature file

❌  Will break downstream classpath imports due to folder rename

### CODE REVIEW

File: catalog/etcd/etcd.bom [hunk 1]
```diff
@@ -1,6 +1,6 @@
 brooklyn.catalog:
   version: "2.4.0-SNAPSHOT" # BROOKLYN_ETCD_VERSION
-  iconUrl: https://raw.githubusercontent.com/coreos/etcd/master/logos/etcd-glyph-color.png
+  iconUrl: https://raw.githubusercontent.com/coreos/etcd/master/logos/etcd-stacked-color.png
```
Comment (grkvlt) [inline catalog/etcd/etcd.bom]: see previous comment on icon

File: catalog/etcd/catalog.bom [hunk 2]
```diff
@@ -0,0 +1,37 @@
+brooklyn.catalog:
+  items:
+    - classpath://io.brooklyn.etcd.brooklyn-etcd:etcd/etcd.bom
+
+    - id: etcd-cluster-template
+      name: "etcd-cluster"
+      iconUrl: https://raw.githubusercontent.com/coreos/etcd/master/logos/etcd-stacked-color.png
```
Comment (grkvlt) [inline catalog/etcd/catalog.bom]: Can this file be copied to `resources/icons/etcd.png` in the repo, and referenced as a `classpath://` resource instead, to allow use when github access is not possible.

**Review by grkvlt — CHANGES_REQUESTED** [review]
Looks good, only change is to copy icon file locally

**Comment by m4rkmckenna** [comment]
@grkvlt I put the etcd icon on the classpath

**Review by Graeme-Miller — APPROVED** [review]

**Review by grkvlt — APPROVED** [review]
Thanks @m4rkmckenna happy with this

**Pull request merged**